src/main/java/org/broadinstitute/hellbender/utils/activityprofile/BandPassActivityProfile.java

```diff
@@ -144,14 +144,14 @@ public double getSigma() {
         for ( final ActivityProfileState superState : super.processState(justAddedState) ) {
             if ( superState.isActiveProb() > 0.0 ) {
                 for( int i = -filterSize; i <= filterSize; i++ ) {
-                    final SimpleInterval loc = getLocForOffset(justAddedState.getLoc(), i);
+                    final SimpleInterval loc = getLocForOffset(superState.getLoc(), i);
                     if ( loc != null ) {
                         final double newProb = superState.isActiveProb() * gaussianKernel[i + filterSize];
```

Comment:
Why in the world is this method still called isActiveProb() when it returns a double? I suppose this isn't your problem but this code could use some work